contracts/lib/EncodeLib.sol

```diff
@@ -13,6 +13,15 @@ library EncodeLib {
 
     error ChainIdAndHashesLengthMismatch(uint256 chainIdsLength, uint256 hashesLength);
 
+    function getSender(PackedUserOperation calldata userOp) internal pure returns (address) {
```

Comment:
changing to msg sender will be easier

Reply:
Rly? I assumed using msg.sender is both cheaper and less bytecode

Reply:
yea. but somewhere we have to check if msg.sender == userop.sender